tests/testthat/test_util.R

```diff
@@ -836,3 +836,11 @@ test_that("get_mode", {
   res <- get_mode(x, na.rm = TRUE)
   expect_equal(as.character(res), "2019-01-02")
 })
+
+test_that("summarize_group", {
+ df <- mtcars %>% exploratory::summarize_group(grp_cols = c("cyl", "mpg"), grp_aggregations = c("none", "mean"), count = n())
```

Comment:
grp_aggregations argument is not really about aggregation. It's rather categorization.
Also, abbreviating "group" to "grp" only in the argument names while the function name is summarize_group looks a little inconsistent to me. How about "group_cols" and "group_funs" instead of "grp_cols" and "grp_aggregations"?

Reply:
Changed argument name as suggested.